content/influxdb/v2/install/_index.md

```diff
@@ -183,177 +190,101 @@ or rename them before putting them in your `$PATH`.
+   ```
 
-    ```sh
-    # Red Hat/CentOS/Fedora AArch64 (ARMv8-A)
-    curl -O https://dl.influxdata.com/influxdb/releases/influxdb2-{{< latest-patch >}}-1.aarch64.rpm
-    sudo yum localinstall influxdb2-{{< latest-patch >}}-1.aarch64.rpm
-    ```
+   ```sh
+   # Red Hat/CentOS/Fedora AArch64 (ARMv8-A)
+   curl -O https://dl.influxdata.com/influxdb/releases/influxdb2-{{< latest-patch >}}-1.aarch64.rpm
+   sudo yum localinstall influxdb2-{{< latest-patch >}}-1.aarch64.rpm
+   ```
 
-2.  Start the InfluxDB service:
+2. Start the InfluxDB service:
 
-    <!--pytest.mark.skip-->
+   <!--pytest.mark.skip-->
 
-    ```sh
-    sudo service influxdb start
-    ```
+   ```sh
+   sudo service influxdb start
+   ```
 
-    Installing the InfluxDB package creates a service file at `/lib/systemd/system/influxdb.service`
-    to start InfluxDB as a background service on startup.
+   Installing the InfluxDB package creates a service file at `/lib/systemd/system/influxdb.service`
+   to start InfluxDB as a background service on startup.
 
-3. Restart your system and verify that the service is running correctly:
+3. To verify that the service is running correctly, restart your system and then enter the following command in your terminal:
 
-    ```
-    $  sudo service influxdb status
-    ● influxdb.service - InfluxDB is an open-source, distributed, time series database
+   ```sh
+   sudo service influxdb status
+   ```
+
+   If successful, the output is the following:
+
+   ```text
+   ● influxdb.service - InfluxDB is an open-source, distributed, time series database
       Loaded: loaded (/lib/systemd/system/influxdb.service; enabled; vendor preset: enable>
       Active: active (running)
-    ```
+   ```
 
 For information about where InfluxDB stores data on disk when running as a service,
 see [File system layout](/influxdb/v2/reference/internals/file-system-layout/?t=Linux#installed-as-a-package).
 
-To customize your InfluxDB configuration, use either
-[command line flags (arguments)](#pass-arguments-to-systemd), environment variables, or an InfluxDB configuration file.
-See InfluxDB [configuration options](/influxdb/v2/reference/config-options/) for more information.
+#### Pass configuration options to the service
 
-#### Pass arguments to systemd
+You can use systemd to customize [InfluxDB configuration options](/influxdb/v2/reference/config-options/#configuration-options) and pass them to the InfluxDB service.
 
-1. Add one or more lines like the following containing arguments for `influxd` to `/etc/default/influxdb2`:
+1. Edit the `/etc/default/influxdb2` service config file to assign variable names to `influxd` command line flags--for example, add one or more `<VARIABLE_NAME>=<COMMAND_LINE_FLAG>` lines like the following:
 
    <!--pytest.mark.skip-->
 
    ```sh
    ARG1="--http-bind-address :8087"
-   ARG2="<another argument here>"
+   ARG2="--storage-wal-fsync-delay=15m"
    ```
```

Comment:
```suggestion
   ```

   When the `influxdb2` service starts, systemd creates environment variables from the directives you set in `/etc/default/influxdb2` and makes the variables available to the service.
```